src/main/python/image_preprocessing/face_manager.py

```diff
@@ -25,6 +25,30 @@ def __init__(self, shape_predictor_path):
         """
         self.shape_predictor_path = shape_predictor_path
 
+    def face_likeness_extraction(self, image):
+        """
+        This function find a face in the image passed and is optimised
+        to align the face before being returned.
+        Author(s):
+            Stephan Nell
+        Args:
+            image (:obj:'OpenCV image'): Image containing the face we need to detect and extract.
+        Raises:
+            ValueError: If no face can be detected.
+        Returns:
+            obj:'OpenCV image': Any background and unnecessary components are removed and only
+            the aligned face is returned in gray scale for facial likeness.
+        Todo:
+            Return error if no face detected
+
+        """
+        rectangle = self.detect(image)
+        predictor = dlib.shape_predictor(self.shape_predictor_path)
+        face_aligner = FaceAligner(predictor, desiredFaceWidth=256)
+        face_aligned = face_aligner.align(image, image, rectangle)
+        gray_face = cv2.cvtColor(face_aligned, cv2.COLOR_BGR2GRAY)
```

Comment:
If we remove the grey scale from the function it will be an exact copy of extract_face the reason we decided on this was speed related if we use extract_face we are using unnecessary additional steps to extract the face for face_verify so we decided on this function.

I also don't totally agree that people will think that this is some sort of side effect since most of the project works on a similar basis. However, if you believe we need to return the RGB image as well to reduce confusion I can return the RGB image and the Grey image as a tuple like in a similar fashion we do with face_verify